src/sanitization.js

```diff
@@ -22,7 +22,6 @@ if (__NODE__) {
     '<': '\\u003C',
     '>': '\\u003E',
     '"': '\\u0022',
```

Comment:
Should we also be escaping single quotes?

Reply:
| Library | `'` | `&` |
|---------|-----|-----|
| [React](https://github.com/facebook/react/blob/1034e26fe5e42ba07492a736da7bdf5bf2108bc6/packages/react-dom/src/server/escapeTextForBrowser.js)   | ✅   | ✅   |
| [Vue](https://github.com/vuejs/vue/blob/02f8b806768d70c589e646c384e592e93387b994/src/platforms/web/server/util.js#L38-L42)     | ❌   | ✅   |
| [Preact](https://github.com/developit/preact-render-to-string/blob/6e2a9e4b47d56060459c409cad18f14c8da47611/src/util.js)
  | ❌   | ✅   |

Based on this sophisticated cargo-cult analysis, it seems we should escape ampersand, but not single quotes.